upcase_booklist_app/templates/category.html

```diff
@@ -0,0 +1,37 @@
+<html>
+    <head>
+        <title>Curious Programmer</title>
+        <link rel="stylesheet" href="{{ url_for('static', filename='main.css') }}">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+       <nav class="page-header">
+           <div class="header-items"><a href="{{ url_for('book_list') }}">Books</a></div>
```

Comment:
One refactor you should do in the next week or two is renam the `book_list` view function to `books`, to match the ones for authors and lists.